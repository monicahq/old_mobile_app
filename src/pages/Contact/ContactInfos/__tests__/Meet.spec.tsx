import {contact1} from '@src/__mock__/contact';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import {Meet} from '../Meet';

describe('Pages', () => {
  describe('Contact / Meet', () => {
    it('should renders correctly', () => {
      expect(toJson(shallow(<Meet contact={contact1} />))).toMatchSnapshot();
    });
  });
});
