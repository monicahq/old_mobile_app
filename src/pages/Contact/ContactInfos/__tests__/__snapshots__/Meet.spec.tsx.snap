// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Contact / Meet should renders correctly 1`] = `
<ForwardRef(View)
  style={
    Object {
      "marginTop": 5,
    }
  }
>
  <ListItem
    key="0"
    lastItem={false}
    title="12/17/1991"
  />
  <ListItem
    key="1"
    lastItem={true}
    title="test"
  />
</ForwardRef(View)>
`;
